ament_cmake_pep257/doc/index.rst

```diff
@@ -0,0 +1,42 @@
+ament_pep257
+============
+
+Checks the code style of Python source files using `pep257
+<http://pep257.readthedocs.org/>`_.
+Files with the following extensions are being considered: ``.py``.
+
+
+How to run the check from the command line?
+-------------------------------------------
+
+The command line tool is provided by the package `ament_pep257
+<https://github.com/ament/ament_lint>`_.
+
+
+How to run the check from within a CMake ament package as part of the tests?
+----------------------------------------------------------------------------
+
+``package.xml``:
+
+.. code:: xml
+
+    <build_depend>ament_cmake_test</build_depend>
```

Comment:
Why dose `ament_cmake_test` need to be a build dependency? I looked for documentation in the `ament_cmake_test` (https://github.com/ament/ament_cmake/tree/master/ament_cmake_test) package but didn't find anything about it.